deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -0,0 +1,691 @@
+    ...getPoolRegistryPermissions(ARBITRUMONE_POOL_REGISTRY),
+    ...getPrimePermissions(ARBITRUMONE_PRIME),
+    ...getPrimeLiquidityProviderPermissions(ARBITRUMONE_PLP),
+    ...getProtocolShareReservePermissions(ARBITRUMONE_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsTimebased(),
+    ...getIRMPermissions(),
+  ],
+  ethereum: [
+    ...getResilientOraclePermissions(ETHEREUM_RESILIENT_ORACLE),
+    ...getChainlinkOraclePermissions(ETHEREUM_CHAINLINK_ORACLE),
+    ...getRedstoneOraclePermissions(ETHEREUM_REDSTONE_ORACLE),
+    ...getBoundValidatorPermissions(ETHEREUM_BOUND_VALIDATOR),
+    ...getSFrxETHOraclePermissions(ETHEREUM_sFrxETH_ORACLE),
+    ...getXVSPermissions(ETHEREUM_XVS),
+    ...getXVSBridgeAdminPermissions(ETHEREUM_XVS_BRIDGE_ADMIN),
+    ...getXVSVaultPermissions(ETHEREUM_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(ETHEREUM_POOL_REGISTRY),
+    ...getPrimePermissions(ETHEREUM_PRIME),
+    ...getPrimeLiquidityProviderPermissions(ETHEREUM_PLP),
+    ...getProtocolShareReservePermissions(ETHEREUM_PSR),
+    ...getConverterNetworkPermissions(ETHEREUM_CONVERTER_NETWORK),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsBlockbased(),
+    ...getIRMPermissions(),
+    ...getConverterPermissions(),
+    ...getXVSVaultTreasuryPermissions(ETHEREUM_XVS_VAULT_TREASURY),
+  ],
+  opbnbmainnet: [
+    ...getResilientOraclePermissions(OPBNBMAINNET_RESILIENT_ORACLE),
+    ...getBoundValidatorPermissions(OPBNBMAINNET_BOUND_VALIDATOR),
+    ...getBinanceOraclePermissions(OPBNBMAINNET_BINANCE_ORACLE),
+    ...getXVSPermissions(OPBNBMAINNET_XVS),
+    ...getXVSBridgeAdminPermissions(OPBNBMAINNET_XVS_BRIDGE_ADMIN),
+    ...getXVSVaultPermissions(OPBNBMAINNET_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(OPBNBMAINNET_POOL_REGISTRY),
+    ...getProtocolShareReservePermissions(OPBNBMAINNET_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getIRMPermissions(),
+  ],
+  arbitrumsepolia: [
+    ...getResilientOraclePermissions(ARBITRUMSEPOLIA_RESILIENT_ORACLE),
+    ...getChainlinkOraclePermissions(ARBITRUMSEPOLIA_CHAINLINK_ORACLE),
+    ...getRedstoneOraclePermissions(ARBITRUMSEPOLIA_REDSTONE_ORACLE),
+    ...getBoundValidatorPermissions(ARBITRUMSEPOLIA_BOUND_VALIDATOR),
+    ...getXVSPermissions(ARBITRUMSEPOLIA_XVS),
+    ...getXVSBridgeAdminPermissions(ARBITRUMSEPOLIA_XVS_BRIDGE_ADMIN),
+    ...getXVSVaultPermissions(ARBITRUMSEPOLIA_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(ARBITRUMSEPOLIA_POOL_REGISTRY),
+    ...getPrimePermissions(ARBITRUMSEPOLIA_PRIME),
+    ...getPrimeLiquidityProviderPermissions(ARBITRUMSEPOLIA_PLP),
+    ...getProtocolShareReservePermissions(ARBITRUMSEPOLIA_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsTimebased(),
+    ...getIRMPermissions(),
+  ],
```

Comment:
Could you also add commands to revoke Multichain governance permission on all testnets to sync them with mainnets?